fec_eregs/static/fec_eregs/js/templates/nav-about.hbs

```diff
@@ -0,0 +1,27 @@
+<div class="mega-container">
+  <div class="mega mega--secondary">
+    <div class="mega__inner">
+      <div class="row">
+        <div class="usa-width-one-fourth">
+          <h2 class="mega-heading__title icon-heading--data-flag-circle--secondary"><a href="/about/">Learn more about the FEC</a></h2>
+        </div>
+        <div class="usa-width-one-half mega__intro">
+          <ul class="usa-width-one-third">
+           <li class="mega__item"><a href="/updates">News and announcements</a></li>
```

Comment:
https://github.com/fecgov/fec-eregs/blob/bac16a529e19bc423fbbfd696192860d78d13678/fec_eregs/static/fec_eregs/js/templates/nav-about.hbs#L10-L21

Should these use the `FEC_CMS_URL` parameter like the main header? I'm getting an error when I click on the links.

Reply:
<b>Navigation links using the env var</b>
The reason that the navigation links do not even render the FEC_CMS_URL env var is because the eRegs navigation templates are currently handlebars templates. So `{{ some_variable }}` in a handlebars template is looking for a var defined in a HB helper or somewhere else in the JS, whereas  {{ some_variable }} in a django template is looking for a va defined in the Django app.

We use relative URLS on the fec-cms for these menu links but perhaps, as you mention,  these should be using the FEC_CMS_URL env var in eRegs since it is really a separate app and other links in here use that variable. 

Note: I do have a follow up WIP-PR to switch switch from using HBS templates to Dango partials like we do on fec-cms because I believe the use of HBS templates here is no longer necessary and is just a residual of an older structure that was just never updated on the eRegs side. This would allow us to use the vars in the links.

<b>For testing  links locally:</b>
The reason links in the main header links (calendar link and logo link to home) don't work locally is because the FEC_CMS_URL env var is  set to point to :8000 which is the same port eRegs runs on locally. 

One  could run eRegs on port :3000 and then run your local CMS on :8000 to test those links
or one can also change settings in `fec-eregs/local_settings.py` to set local vars that override whats in the settings/* files for testing locally. (Like pointing the FEC_CMS_URL var to the dev cms site as @lbeaufort had mentioned

Reply:
Thanks for your research on this! I really appreciate it.